docroot/themes/custom/uids_base/scss/components/page-layouts.scss

```diff
@@ -24,6 +24,157 @@
 .page__container.page__container--full {
   max-width: 100%;
 }
+html {
+  scroll-behavior: smooth;
+}
+
+@media (prefers-reduced-motion: reduce) {
+  html {
+    scroll-behavior: auto;
+  }
+}
+.scrollUp {
+  position: fixed;
+  z-index: 2;
+  position: fixed;
+  right: -20px;
+  height: 100vh;
+  top: 0;
+  padding-right: 20px;
+  align-items: center;
```

Comment:
I'd also set the bottom link to appear towards the bottom of the page:
```
bottom: 10vh;
align-items: self-end;
```

Reply:
@bspeare THANK YOU! I'll change that!